### ISSUE TITLE
Got same problem that model only return lots of  '\n'

### ISSUE THREAD

**Issue by FoxxComz** [ise-uiuc/magicoder#30]
I got the same problem when use the quick start script to run an inference task.
Just the same as this issue: https://github.com/ise-uiuc/magicoder/issues/22
what should i do to solve this problem?

**Comment by FoxxComz** [comment]
here is my code:
`from transformers import pipeline
import torch

MAGICODER_PROMPT = """You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
{instruction}

@@ Response
"""

instruction = "Implement a high-level API for a TODO list application. The API takes as input an operation request and updates the TODO list in place. If the request is invalid, raise an exception."

model_path='/data/MagiCoder'
prompt = MAGICODER_PROMPT.format(instruction=instruction)
generator = pipeline(
    model=model_path,
    task="text-generation",
    torch_dtype=torch.bfloat16,
    device_map="auto",
)
result = generator(prompt, max_length=2048, num_return_sequences=1, temperature=0.0)
print(result[0]["generated_text"])`

**Comment by UniverseFly** [comment]
Thanks for reporting this. Did you also try to update the package as described in the issue you referred to, and could that help?

**Comment by FoxxComz** [comment]
> Thanks for reporting this. Did you also try to update the package as described in the issue you referred to, and could that help?

do u have recomended version of packages that magicoder need?

**Comment by UniverseFly** [comment]
Yes they are listed in [pyproject.toml](https://github.com/ise-uiuc/magicoder/blob/c900e74a577ea06d3a397b34745d0c920a91f1f7/pyproject.toml).
